src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Feedback/FeedbackFileLoggerProviderFactory.cs

```diff
@@ -5,6 +5,10 @@ namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Feedback
 {
     internal abstract class FeedbackFileLoggerProviderFactory
     {
-        public abstract FeedbackFileLoggerProvider GetOrCreate();
+        /// <summary>
+        /// Returns a <see cref="FeedbackFileLoggerProvider"/>. This is an optimization to ensure we don't load an extra logging dll at MEF load time in Visual Studio.
```

Comment:
"This is an optimization" - doesn't really convey what part of this is an optimization. Would be good to clarify that it returns object because blah... in `<remarks>` section.